client/homebrew/pages/basePages/listPage/brewItem/brewItem.jsx

```diff
@@ -90,11 +91,15 @@ const BrewItem = createClass({
 		</a>;
 	},
 
-	renderGoogleDriveIcon : function(){
-		if(!this.props.brew.googleId) return;
+	renderStorageIcon : function(){
+		if(!this.props.brew.googleId) return <span title='Internal Homebrewery Database'>
+			<img className='homebreweryIcon' src={homebreweryIcon} alt='homebreweryIcon' />
+		</span>;
 
-		return <span>
-			<img className='googleDriveIcon' src={googleDriveIcon} alt='googleDriveIcon' />
+		return <span title='Google Drive Storage'>
```

Comment:
Maybe the Title should be “Saved on [User]’s Google Drive”?

Reply:
This could be done with the `owners` field from the Google API. Unfortunately it would only work for brews owned by the current logged-in user where we call `listGoogleBrews()`

Maybe in that case we could display `Another user's Google Drive Storage` to at least make clear that you aren't logged in to the owning account.